python/cuml/fil/README.md

```diff
@@ -33,17 +33,18 @@ Additionally, FIL can be called directly from C or C++ code. See [the API docs h
 # Features
 
 * Input model source: XGBoost (binary format), cuML RandomForest, scikit-learn RandomForest, LightGBM
-* Model types: Regression, Binary Classification, Multi-class Classification (for cuML Random Forests, but not GBDTs or scikit-learn Random Forests)
+* Model types: Regression, Binary Classification, Multi-class Classification (for cuML Random Forests or GBDTs, but not scikit-learn Random Forests)
```

Comment:
If we're updating the documentation, I think the following would be less ambiguous:

Model types:
  - Regression (all input models)
  - Binary classification (all input models)
  - Multi-class classification (cuML RF, GBDT, but not SciKit-Learn RF)